---
bool detected	#gefunden
string color	#farbe
